processors/HNL.py

```diff
@@ -121,8 +121,9 @@
 
 analyzerChain.append(
```

Comment:
This module is now redundant because you have the same one "nominal" in the loop

Reply:
Put every other module that depends on the jets also in the loop, like the LepJet selection